packages/node_modules/@webex/internal-plugin-metrics/src/call-diagnostic-events-batcher.js

```diff
@@ -23,6 +23,7 @@ const CallDiagnosticEventsBatcher = Batcher.extend({
   prepareRequest(queue) {
     // Add sent timestamp
     queue.forEach((item) => {
+      item.eventPayload.originTime = item.eventPayload.originTime || {};
```

Comment:
Since this is a change to a separate plugin, it needs to be another commit please.

Reply:
done: https://github.com/webex/webex-js-sdk/pull/1527